EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+import typing
+
+# time constants in seconds
+MINUTE = 60
+HOUR = MINUTE * 60
+DAY = HOUR * 24
+WEEK = DAY * 7
+
+
+class Task:
+    def __init__(self, wrapped_func, duration: int, max_runs: int):
```

Comment:
`wrapped_func` is not typed

Reply:
I'll change it to a `Coroutine`, which I plan on being inside of the `types.py` file.